usaspending_api/common/query_with_filters.py

```diff
@@ -166,12 +166,11 @@ class _RecipientSearchText(_Filter):
     @classmethod
     def generate_elasticsearch_query(cls, filter_values: List[str], query_type: _QueryType) -> ES_Q:
         recipient_search_query = []
-        fields = ["recipient_name"]
 
         for v in filter_values:
             upper_recipient_string = es_sanitize(v.upper())
             recipient_name_query = ES_Q(
-                "query_string", query=upper_recipient_string + "*", default_operator="AND", fields=fields
+                "match", recipient_name={"query": upper_recipient_string + "*", "operator": "AND"}
```

Comment:
I don't think this is working as expected. The `match` query is going to look for a literal `*` since it doesn't support the same query_string syntax. I haven't tried the code locally, but trying a `match` against my local ES cluster and it did not support wildcard character.

Reply:
Saw you changed it, but just to add to this the tests were still passing because apparently the wildcard character is simply ignored. Query for `Recipient_name_1*` yields transactions with `Recipient_name_1` and not `Recipient_name_10`.

Reply:
We should probably have tests that check the wildcard is working - I believe we have some on the keyword filter for advanced search, so it's odd that the recipient filter doesn't check that as well.